### PR TITLE
Chapter 9: Advanced login

### PR DESCRIPTION
**Pull request by hlongdesu**
Screenshot from 2019-05-30 16-30-14

### CODE REVIEW

**Comment by hlongdesu** [comment]
ready

**Comment by hlongdesu** [comment]
ready

File: test/test_helper.rb [hunk 1]
```diff
@@ -8,4 +8,16 @@ class ActiveSupport::TestCase
   def is_logged_in?
     !sessions[:user_id].nil?
```
Comment (VuThiTranVan) [inline test/test_helper.rb]: dùng present? đi gái

**Comment by VuThiTranVan** [comment]
merged
có 1 comment nhỏ, fix hì

**Pull request merged**